openedx/adg/lms/bootcamps/tests/test_helpers.py

```diff
@@ -0,0 +1,53 @@
+"""
+Tests for all the code in templatetags/helpers.py Bootcamps application.
+"""
+import mock
+
+from openedx.adg.lms.bootcamps.templatetags.constants import (
+    NOT_STARTED,
+    IN_PROGRESS,
+    COMPLETED
+)
+from openedx.adg.lms.bootcamps.templatetags.helpers import (
+    is_user_enrolled_in_course,
+    get_course_progress_for_user,
+)
+from student.tests.factories import UserFactory, CourseEnrollmentFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+
+class HelpersTestCase(ModuleStoreTestCase):
+    """
+    TestCase class for the helpers.py functions
+    """
+    USERNAME = "test"
+    EMAIL = "test@example.com"
+    PASSWORD = "edx"
```

Comment:
the same data is created in `test_bootcamp_extras.py`. Let's move it to a `constants.py`.